### DISCUSSION TITLE
Installing "Golf" release to internal SATA SSD drive is not bootable on Beelink U59 Pro

### DISCUSSION THREAD

**Discussion by brandonb927** [kg4vdk/arcos-linux-modules#39] [General Discussion]
Hey there, I am having a heck of a time trying to get my machine to boot from the internal SSD after installing arcOS to it from the USB. The BIOS will not recognize the bootable partition evne when CD/DVD and USB are the only bootable options in the system. The only way I can boot the internal SSD is by having the arcOS USB drive plugged in as you mention here https://youtu.be/xe6mucTuQF4?feature=shared&t=1727

Any thoughts on why this might be? I have tried disabling and re-enabling Secure Boot (I know that shouldn't matter but thought I'd try) and I've tried almost every permutation of bootable options to try to get the SSD-installed arcOS installation to boot without the USB plugged in but my machine just simply will not boot the internal drive partition without the USB installed.

**Comment by kg4vdk** [comment]
Hey Brandon,

A few questions...

Can you walk me through how you wrote the ISO to the internal drive?

> The BIOS will not recognize the bootable partition evne when CD/DVD and USB are the only bootable options in the system.

Does this mean that in the BIOS you have CD/DVD and/or USB devices set as the primary boot device? If so, that should be changed to point to the internal disk.

> The only way I can boot the internal SSD is by having the arcOS USB drive plugged in

Can you elaborate on this? I'm wondering what you're seeing that indicates that you're booting to the internal SSD with the USB attached.

The ISO is bootable in UEFI or CSM (legacy) mode, so if that's a setting you can change, it shouldn't have any bearing. I know you didn't mention it, but thought ruling it out wouldn't hurt. :-)

If you don't mind, can you also provide a full system report? You can attach one here (or just provide the link) by using the "System Info" utility from the Main Menu, and selecting "Upload system information."

73 de KG4VDK,
Mike

**Reply by brandonb927** [reply]
> Can you walk me through how you wrote the ISO to the internal drive?

I followed your directions in the originally linked YouTube video by writing the ISO disk image (which I downloaded onto the USB stick from the download on the site) to the unformatted SSD using the Disks utility, following the exact steps laid out in said video. 

> Does this mean that in the BIOS you have CD/DVD and/or USB devices set as the primary boot device? If so, that should be changed to point to the internal disk.
> Can you elaborate on this? I'm wondering what you're seeing that indicates that you're booting to the internal SSD with the USB attached.

As soon as arcOS is installed onto the SSD and I reboot, that SSD disappears from my BIOS as a bootable disk completely. It's only when I have the USB drive with arcOS installed plugged into the machine that the bootable drive appears in my BIOS boot options. If I run the "Wipe arcOS Install" utility and reboot back into my BIOS, _only then without the USB drive plugged in_ does the SSD reappear in my boot options; it's very strange, I've never seen that happen before and I've installed linux distros on several computers. I will attach some images of this sequence of events in another comment.

System report: https://termbin.com/27ai

**Reply by brandonb927** [reply]
My machine has a 512GB NVMe SSD installed and a 128gb Samsung SATA SSD (where I'm installing arcOS).

Boot options list with (first) and without (second) the arcOS USB drive plugged in
<img src="https://github.com/user-attachments/assets/f8dbd9f6-d869-4477-9f38-fa2a3a9eef46" width=300 />
<img src="https://github.com/user-attachments/assets/097ed5ce-bc52-4c9f-90db-a6ed3f06abab" width=300 />

BIOS options with (first) and without (second) the arcOS USB drive plugged in
<img src="https://github.com/user-attachments/assets/fd75a373-6a62-4d4b-aad7-7a26dae4e940" width=300 />
<img src="https://github.com/user-attachments/assets/6ca3716b-25c7-49a7-9391-e4ab992fae47" width=300 />

Wiping the arcOS install and rebooting to BIOS yields the following options with the SSD
<img src="https://github.com/user-attachments/assets/0417a301-42dc-42cb-9a13-69baf19b97fe" width=300 />
<img src="https://github.com/user-attachments/assets/b12a483a-fd15-452f-9322-790bf1f96f83" width=300 />

**Reply by kg4vdk** [reply]
Thanks for providing the screen shots! I'm thinking that there's something up with Windows Boot Manager still being in the mix. Is the 512GB NVMe disk a Windows install? If there is no Windows install on the machine, that Windows Boot Manager entry needs to be deleted/disabled.

**Reply by brandonb927** [reply]
The more I futz around with this the more I'm thinking this might just be a quirk of my machine: I disabled the Windows disk from my boot menu completely and the machine gets stuck in a reboot loop trying to find a bootable device. I'm wondering if I should just use a USB drive as the bootable for the internal drive then? That seems to be the only way to get it to boot arcOS from the SSD.

**Reply by kg4vdk** [reply]
I did some forum searching regarding "boot from SATA" on the Beelink U59, and it doesn't seem like you are alone in this issue. They look like pretty cool devices, but this does appear to be a limitation they have. Some folks reported, or suggested, being able to mark the secondary SATA drive as bootable in BIOS. But, I didn't find any reliable methods for doing this to pass along to you. :-(

**Reply by brandonb927** [reply]
Dang, I hadn't gone on the quest to search the internet just yet. I thought this might've just been a quirk with arcOS and the way it boots because I was able to install and boot Ubuntu previously from the SATA SSD 🤷 I think the issue I'm having might be that arcOS isn't booting in UEFI mode (at least on this device, I know you mentioned it can boot in UEFI mode previously)? Anyways, I appreciate your sleuthing and this is not the end of the world for me as I have another machine on the way from eBay that I may use solely for arcOS instead. Cheers!
